core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -287,6 +287,28 @@ private[spark] class Executor(
       notifyAll()
     }
 
+    /**
+     *  Utility function to:
+     *    1. Report executor runtime and JVM gc time if possible
+     *    2. Collect accumulator updates
+     *    3. Set the finished flag to true and clear current thread's interrupt status
+     */
+    private def collectAccumulatorsAndResetStatusOnFailure(taskStart: Long) = {
+      // Report executor runtime and JVM gc time
+      Option(task).foreach(t => {
+        t.metrics.setExecutorRunTime(System.currentTimeMillis() - taskStart)
```

Comment:
`taskStartTime`

Reply:
e, `startStart` is already defined previously.  Do you think we need to replace all the `taskStart` to `taskStartTime`

Reply:
we should at least rename it in this method as it's newly added code. We can also update the existing code if it's not a lot of work.

Reply:
Will do it..